pythran/pythonic/types/numpy_texpr.hpp

```diff
@@ -200,5 +200,15 @@ struct __combined<pythonic::types::numpy_texpr<E>, K> {
     typedef pythonic::types::numpy_texpr<E> type;
 };
 
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::numpy_texpr<E>, pythonic::types::ndarray<T, N>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
+
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::ndarray<T, N>, pythonic::types::numpy_texpr<E>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
```

Comment:
there's the same kind of issue as the one I already spotted for gexpr here, even if my solution has some flaws.